src/open_inwoner/js/components/search/index.js

```diff
@@ -11,15 +11,17 @@ const radioButtons = document.querySelectorAll(
   })
 })
 
-var timerId = 0
+let timerId = null
 
 const searchForm = document.getElementById('search-form')
 
 const filterButtons = document.querySelectorAll('.filter .checkbox__input')
 ;[...filterButtons].forEach((checkbox) => {
   checkbox.addEventListener('change', (event) => {
     clearInterval(timerId)
```

Comment:
This should be `clearTimeout(timerId)`. Although this works, from MDN:

> It's worth noting that the pool of IDs used by [setInterval()](https://developer.mozilla.org/en-US/docs/Web/API/setInterval) and [setTimeout()](https://developer.mozilla.org/en-US/docs/Web/API/setTimeout) are shared, which means you can technically use clearInterval() and [clearTimeout()](https://developer.mozilla.org/en-US/docs/Web/API/clearTimeout) interchangeably. However, for clarity, you should avoid doing so.

Reply:
Oops, let's completely get rid of that interval....